### PR TITLE
Added cra s3 deploy instructions

### PR DESCRIPTION
**Pull request by juanisierra**
* Added instructions for deploying create-react-app project to s3.

### CODE REVIEW

File: frontend/docs/scripts/deploy-s3.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  const headers = {
+    'Content-Encoding': 'gzip'
+  };
+
+  const zipped = gulp.src(zippedFiles).pipe(awspublish.gzip({ ext: '' })).pipe(publisher.publish(headers));
+  const unzipped = gulp.src(unzippedFiles).pipe(publisher.publish());
+  return merge(zipped, unzipped).pipe(publisher.sync()).pipe(awspublish.reporter());
+};
+
+gulp.task('publish-stg', () => {
```
Comment (mvbattan) [inline frontend/docs/scripts/deploy-s3.js]: I'd extract this arrow function inside a parametrizable function, so we can reuse it in `publish-prod` task.

File: frontend/docs/cra-s3-compressed-deploy.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Deploy Create React App project to S3
+
+## Dependencies
+To install them just run
+```
+npm install --save gulp gulp-awspublish aws-sdk merge-stream
+```
+## Configuration
```
Comment (SKOLZ) [inline frontend/docs/cra-s3-compressed-deploy.md]: great!!!

File: frontend/docs/cra-s3-compressed-deploy.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+* Copy the [deploy script](./scripts/deploy-s3.js) inside the project's directory. The recommended route is `./gulp/tasks/deploy-s3.js`
+
+* Create `gulpfile.js` if the project wasn't using gulp alredy or else edit `gulpfile.js` and add:
+```
+require('./gulp/tasks/deploy-s3.js')
+```
+* Edit `s3-deploy.js` changing `S3_STG_BUCKET_NAME` and `S3_PROD_BUCKET_NAME` for the env variables containing both the staging and production s3 bucket names. Also change `STAGING-DEPLOY-PROFILE` and `PRODUCTION-DEPLOY-PROFILE` for the staging and production deploy aws profile names.
```
Comment (SKOLZ) [inline frontend/docs/cra-s3-compressed-deploy.md]: is it `s3-deploy.js` or `deploy-s3.js` in lime 9 and 13 says the latter one

**Comment by mvbattan** [comment]
@SKOLZ Ping here !

**Pull request merged**